### PR TITLE
Fix #123 : updated Makefile to support windows

### LINKED ISSUE #123: Make file execution failing for windows.
### Environment

- `drf-user` version(s) (`drf-user --version`:
- Operating System(s): Windows
- Python version(s): 3.8

### Description of the bug

Make install is working but make test and livehtml is not working.

### What you expected to happen

Tests passing and html page constructed.

### How to reproduce (as minimally and precisely as possible)

After cloning repo and install make using choco and run the make files getting this error

```
make test
${VIRTUAL_ENV-.venv}/bin/python -m pytest --disable-pytest-warnings --ds=tests.settings --cov=drf_user tests/
process_begin: CreateProcess(NULL, ${VIRTUAL_ENV-.venv}/bin/python -m pytest --disable-pytest-warnings --ds=tests.settings --cov=drf_user tests/, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:67: test] Error 2
```

### PR DESCRIPTION
**Pull request by gauravp446**
## Description
Added condition in Makefile to handle for windows environment

## Motivation and Context
Resolves #123

## Have you tested this? If so, how?
Yes i have tested this.
make_test_ss

## Checklist for PR author(s)
- [x] Changes are covered by unit tests (no major decrease in code coverage %).
- [x] All tests pass.
- [x] Docstring coverage is **100%** via `interrogate drf_user` (I mean, we _should_ set a good example :smile:).
- [x] Updates to documentation:
    - [x] Document any relevant additions/changes in `README.md`.
    - [x] Manually update **both** the `README.md` _and_ `docs/index.rst` for any new changes.
    - [x] Any changed/added classes/methods/functions have appropriate `versionadded`, `versionchanged`, or `deprecated` [directives](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded).  Find the appropriate next version in the project's [``__init__.py``](https://github.com/101loop/drf-user/blob/master/drf_user/__init__.py) file.

label=EPAM_INDIA_OSS_CONTRIBUTORS

### CODE REVIEW

**Review by sumit4613 — APPROVED** [review]
Hey, @gauravp446 really sorry man, took this much time to merge this. Got busy at work that's why I forgot about this. 

Thanks for working on that. Really appreciate this. 🎉 
Welcome to drf-user.

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-VIRTUALENV = virtualenv --python=python3
-PYTHON = $(VENV)/bin/python
-VENV := $(shell echo $${VIRTUAL_ENV-.venv})
+VENV = venv
+PYTHON=$(VENV)/bin/python
+
+# make it work on windows too
+ifeq ($(OS), Windows_NT)
+    PYTHON=$(VENV)/Scripts/python
+endif
```
Comment (sumit4613) [inline Makefile:9]: hey @gauravp446 CI pipeline is failing from your changes, do this
```
VENV = venv
PYTHON=$(VENV)/bin/python
```

Add these two lines in the `ifeq` block and revert back these 3 lines, this should fix the issue.
```
VIRTUALENV = virtualenv --python=python3
PYTHON = $(VENV)/bin/python
VENV := $(shell echo $${VIRTUAL_ENV-.venv})
```

**Review by sumit4613 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (gauravp446) [reply Makefile:9]: Hi @sumit4613 within the `ifeq` block it should be :
`
VENV = venv
PYTHON=$(VENV)/bin/python
`
?

**Review by sumit4613 — APPROVED** [review]
Thanks 💯

**Pull request merged**